force_torque_sensor_broadcaster/doc/userdoc.rst

```diff
@@ -35,3 +35,34 @@ An example parameter file for this controller can be found in `the test director
 
 .. literalinclude:: ../test/force_torque_sensor_broadcaster_params.yaml
    :language: yaml
+
+Wrench Transformer Node
+-----------------------
+The package provides a standalone ROS2 node ``wrench_transformer_node`` that transforms wrench messages published by the ``ForceTorqueSensorBroadcaster`` controller to different target frames using TF2. This is useful when applications need force/torque data in coordinate frames other than the sensor frame.
+
+The node subscribes to wrench messages from the broadcaster (either raw or filtered) and publishes transformed versions to separate topics for each target frame.
+
+Usage
+^^^^^
+The wrench transformer node can be launched as a standalone executable:
+
+.. code-block:: bash
+
+   ros2 run force_torque_sensor_broadcaster wrench_transformer_node
+
+Parameters
+^^^^^^^^^^
+The wrench transformer uses the `generate_parameter_library <https://github.com/PickNikRobotics/generate_parameter_library>`_ to handle its parameters. The parameter `definition file for the wrench transformer <https://github.com/ros-controls/ros2_controllers/blob/{REPOS_FILE_BRANCH}/force_torque_sensor_broadcaster/src/wrench_transformer_parameters.yaml>`_ contains descriptions for all the parameters.
+
+Full list of parameters:
+
+.. generate_parameter_library_details:: ../src/wrench_transformer_parameters.yaml
+  parameters_context.yaml
```

Comment:
```suggestion
```
`WARNING: parameters_context: interface_names not found, insert it at the end of the list`